lib/MagicPen.js

```diff
@@ -236,7 +239,7 @@ MagicPen.prototype.removeFormatting = function () {
     this.output.forEach(function (line, index) {
         result.output[index] = normalizeLine(line.map(function (outputEntry) {
             return outputEntry.style === 'text' ?
-                { style: 'text', args: { content: outputEntry.args.content, styles: [] } } :
+                { style: 'text', args: { content: outputEntry.args.content, styles: [] }, pen: outputEntry.pen } :
```

Comment:
Would it be possible to save the `theme` with the entry instead of the pen? I think that would be more to the appropriate.